app/models/need.rb

```diff
@@ -32,11 +34,16 @@ class Need < ApplicationRecord
                    'Other': 'other' }
 
   validates :category, presence: true
+  validates :name, presence: true
+  validate :valid_start_on?
 
-  ASSESSMENT_CATEGORIES = ['phone triage', 'check in'].freeze
-
-  # validates :food_priority, inclusion: { in: %w[1 2 3] }, allow_blank: true
-  # validates :food_service_type, inclusion: { in: ['Hot meal', 'Heat up', 'Grocery delivery'] }, allow_blank: true
+  # rubocop:disable Style/GuardClause, Style/IfUnlessModifier
```

Comment:
maybe this works - then no need to disable the cop.
```
  validate :start_required_for_assessment

  def start_required_for_assessment
    return unless start_on.nil? && ASSESSMENT_CATEGORIES.include?(category.to_s.downcase) ||
                  start_on.nil? && category.nil?

    errors.add(:start_on, 'must be provided')
  end
```

Reply:
thanks, done